### DISCUSSION TITLE
Getting the lastCheckpointdoc.seq before starting the replication

### DISCUSSION THREAD

**Discussion by MaxGeurtsen** [pouchdb/pouchdb#9008] [Q&A]
Is it possible to get the last seq stored locally before starting the replication?

I want to be able to count how much data will be synced from all collections i have created a `RxReplicationState` for, before I start the sync process.
I want to create a progress bar for the syncing process.

**Comment by SourceR85** [comment]
Hi, Since nobody answered for a while, I'll give you an alternate hint:
I use [activeTasks](https://pouchdb.com/api.html#active_tasks) for this purpose.

This API has 3 methods, you have to handle (take a look on the `Real-time updates` section):
- add
- remove
- update

filter the events for `type == "replication"` and track changes for `total_changes` and `changes_done`.
Keep in mind: during the replication, `total_changes` increases as PouchDB gets more information.
